### DISCUSSION TITLE
Support overrides

### DISCUSSION THREAD

**Discussion by siddhpant** [JanDeDobbeleer/oh-my-posh#4725] [Ideas]
I want to use the `emodipt-extend`, but want to override some settings (like removing branch length limitation).

I can copy and modify, but that doesn't get updated.

Please allow overrides so one can have some modifications over default themes.

Related: #4189, #3078

**Comment by JanDeDobbeleer** [comment]
@siddhpant copy paste the theme and reference the location. Themes are only sample configurations.

**Reply by siddhpant** [reply]
The catalogue is too big to call it a *sample* though.

Plus, the *samples* get updated and can have bug fixes etc. when I update posh, so they aren't really treated as samples. A sample configuration in most projects is just one example file.

**Reply by JanDeDobbeleer** [reply]
@siddhpant but they are. Use then as they are, or copy and adjust to your liking.
